pkg/bff/api/v1/api.go

```diff
@@ -118,7 +118,8 @@ type LookupParams struct {
 // also contain an "error" field - the BFF will handle this field by logging the error
 // but will exclude it from any results returned.
 type LookupReply struct {
-	Results []map[string]interface{} `json:"results"`
+	TestNet map[string]interface{} `json:"testnet"`
+	MainNet map[string]interface{} `json:"mainnet"`
 }
```

Comment:
Cool, this makes sense to me since it matches what I've been doing for the other BFF endpoints.